### DISCUSSION TITLE
GeoNature aux TAAF

### DISCUSSION THREAD

**Discussion by ca-rock** [PnX-SI/GeoNature#1850] [Show and tell]
Bonjour à tous,
Je fais ce message pour me présenter : je suis Caroline Kurzrock, j'ai rejoint les TAAF le mois dernier en tant de développeuse géomaticienne, et je suis principalement chargée d'intégrer GeoNature au sein de notre collectivité. J'interviendrai sûrement régulièrement sur ce repo pour échanger avec vous sur nos protocoles et autres problématiques que je rencontrerai. Merci pour votre travail et à bientôt !

**Comment by camillemonchicourt** [comment]
Bienvenue !
